src/lib/draw/route/route_tool.ts

```diff
@@ -52,24 +53,19 @@ export class RouteTool {
       id: "route-points",
       source,
       filter: isPoint,
-      color: [
-        "match",
+      color: matchValue(
```

Comment:
There were a few places where we were using maplibre case or match manually. All using this one helper method now.